pkgs/by-name/me/mealie/package.nix

```diff
@@ -96,42 +109,32 @@ pythonpkgs.buildPythonApplication rec {
 
     substituteInPlace mealie/__init__.py \
       --replace-fail '__version__ = ' '__version__ = "v${version}" #'
-
-    substituteInPlace mealie/services/backups_v2/alchemy_exporter.py \
-      --replace-fail 'PROJECT_DIR = ' "PROJECT_DIR = Path('$out') #"
-
-    substituteInPlace mealie/db/init_db.py \
-      --replace-fail 'PROJECT_DIR = ' "PROJECT_DIR = Path('$out') #"
-
-    substituteInPlace mealie/services/backups_v2/alchemy_exporter.py \
-      --replace-fail '"script_location", path.join(PROJECT_DIR, "alembic")' '"script_location", "${src}/alembic"'
   '';
 
-  postInstall = let
-    start_script = writeShellScript "start-mealie" ''
-      ${lib.getExe pythonpkgs.gunicorn} "$@" -k uvicorn.workers.UvicornWorker mealie.app:app;
-    '';
-    init_db = writeShellScript "init-mealie-db" ''
-      ${python.interpreter} $OUT/${python.sitePackages}/mealie/scripts/install_model.py
-      ${python.interpreter} $OUT/${python.sitePackages}/mealie/db/init_db.py
+  postInstall =
+    let
+      start_script = writeShellScript "start-mealie" ''
+        ${lib.getExe pythonpkgs.gunicorn} "$@" -k uvicorn.workers.UvicornWorker mealie.app:app;
+      '';
+      init_db = writeShellScript "init-mealie-db" ''
+        ${python.interpreter} $OUT/${python.sitePackages}/mealie/scripts/install_model.py
+        ${python.interpreter} $OUT/${python.sitePackages}/mealie/db/init_db.py
+      '';
+    in
+    ''
+      mkdir -p $out/bin $out/libexec
+      rm -f $out/bin/*
+
+      makeWrapper ${start_script} $out/bin/mealie \
+        --set PYTHONPATH "$out/${python.sitePackages}:${pythonpkgs.makePythonPath dependencies}" \
+        --set LD_LIBRARY_PATH "${crfpp}/lib" \
+        --set STATIC_FILES "${frontend}" \
+        --set PATH "${lib.makeBinPath [ crfpp ]}"
+
+      makeWrapper ${init_db} $out/libexec/init_db \
+        --set PYTHONPATH "$out/${python.sitePackages}:${pythonpkgs.makePythonPath dependencies}" \
+        --set OUT "$out"
     '';
-  in ''
-    mkdir -p $out/bin $out/libexec
-    rm -f $out/bin/*
-
-    substitute ${src}/alembic.ini $out/alembic.ini \
-      --replace-fail 'script_location = alembic' 'script_location = ${src}/alembic'
-
-    makeWrapper ${start_script} $out/bin/mealie \
-      --set PYTHONPATH "$out/${python.sitePackages}:${pythonpkgs.makePythonPath dependencies}" \
-      --set LD_LIBRARY_PATH "${crfpp}/lib" \
-      --set STATIC_FILES "${frontend}" \
-      --set PATH "${lib.makeBinPath [ crfpp ]}"
-
-    makeWrapper ${init_db} $out/libexec/init_db \
-      --set PYTHONPATH "$out/${python.sitePackages}:${pythonpkgs.makePythonPath dependencies}" \
-      --set OUT "$out"
-  '';
```

Comment:
That makes a big diff for only formatting, did you use [nixfmt-rfc-style](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=nix+fmt+rfc) on this file ?

Reply:
Yes this change was done with `nixfmt nixpkgs-unstable-2024-12-04`.